packages/world/src/systems/MoveSystem.sol

```diff
@@ -36,4 +36,21 @@ contract MoveSystem is System {
 
     notify(caller, MoveNotification({ moveCoords: newCoords }));
   }
+
+  // 5 bits per direction, packed into a single uint256 (max 51 directions)
+  function moveDirectionsPacked(EntityId caller, uint256 packedDirections, uint8 count) public {
```

Comment:
how many directions could we do if we encoded the count into the packed type? maybe as the first thing

Reply:
49

Reply:
maybe worth it cuz then we get one bytes32 instead of 2 right?